tests/test_equivalence.py

```diff
@@ -28,13 +29,18 @@
 from fasm2bels.fasm2bels import main
 
 
-test_names = ["add32"]
+test_names = ["add32","alu"]
 
 
 def unpack_tar(tar_file):
     tar = tarfile.open(name=tar_file, mode="r:gz")
     tar.extractall(path=os.path.dirname(tar_file))
 
+def create_golden_file(file_name, part):
+    vivado = os.environ('VIVADO_PATH') 
```

Comment:
Not sure why this changed back to use `os.environ` rather than the previous `os.getenv('VIVADO_PATH', None)` and the assertion to check that `vivado` is not `None`